show/main.py

```diff
@@ -66,6 +66,8 @@
 PLATFORM_JSON = 'platform.json'
 HWSKU_JSON = 'hwsku.json'
 PORT_STR = "Ethernet"
+ECMP_CALC = '/usr/bin/ecmp_calc.py'
```

Comment:
As the script will be available only for MLNX devices, please add check, that if the script is not exist, calling the command will return a message that the script \ tool is not supported

Reply:
Add check is script exists.